soh/soh/Enhancements/randomizer/ShuffleFreestanding.cpp

```diff
@@ -43,3 +43,238 @@ void ShuffleFreestanding_OnVanillaBehaviorHandler(GIVanillaBehavior id, bool* sh
+    locationTable[RC_BOTTOM_OF_THE_WELL_MQ_BASEMENT_HALLWAY_LEFT_HEART] =     Location::Collectable(RC_BOTTOM_OF_THE_WELL_MQ_BASEMENT_HALLWAY_LEFT_HEART,     RCQUEST_MQ,      RCTYPE_FREESTANDING,                                                     ACTOR_EN_ITEM00,      SCENE_BOTTOM_OF_THE_WELL,           TWO_ACTOR_PARAMS(1044, -1223),       "MQ Basement Hallway Left Heart",              RHT_BOTTOM_OF_THE_WELL_HEART,                                    RG_RECOVERY_HEART,    SpoilerCollectionCheck::RandomizerInf(RAND_INF_BOTTOM_OF_THE_WELL_MQ_BASEMENT_HALLWAY_LEFT_HEART));
+    locationTable[RC_BOTTOM_OF_THE_WELL_MQ_BASEMENT_HALLWAY_RIGHT_HEART] =    Location::Collectable(RC_BOTTOM_OF_THE_WELL_MQ_BASEMENT_HALLWAY_RIGHT_HEART,    RCQUEST_MQ,      RCTYPE_FREESTANDING,                                                     ACTOR_EN_ITEM00,      SCENE_BOTTOM_OF_THE_WELL,           TWO_ACTOR_PARAMS(1114, -1223),       "MQ Basement Hallway Right Heart",             RHT_BOTTOM_OF_THE_WELL_HEART,                                    RG_RECOVERY_HEART,    SpoilerCollectionCheck::RandomizerInf(RAND_INF_BOTTOM_OF_THE_WELL_MQ_BASEMENT_HALLWAY_RIGHT_HEART));
+
+    locationTable[RC_GANONS_CASTLE_MQ_WATER_TRIAL_HEART] =                    Location::Collectable(RC_GANONS_CASTLE_MQ_WATER_TRIAL_HEART,                    RCQUEST_MQ,      RCTYPE_FREESTANDING,                                                     ACTOR_EN_ITEM00,      SCENE_INSIDE_GANONS_CASTLE,         TWO_ACTOR_PARAMS(1743, -528),        "MQ Water Trial Heart",                        RHT_GANONS_CASTLE_HEART,                                         RG_RECOVERY_HEART,    SpoilerCollectionCheck::RandomizerInf(RAND_INF_GANONS_CASTLE_MQ_WATER_TRIAL_HEART));
+    locationTable[RC_GANONS_CASTLE_MQ_LIGHT_TRIAL_RIGHT_HEART] =              Location::Collectable(RC_GANONS_CASTLE_MQ_LIGHT_TRIAL_RIGHT_HEART,              RCQUEST_MQ,      RCTYPE_FREESTANDING,                                                     ACTOR_EN_ITEM00,      SCENE_INSIDE_GANONS_CASTLE,         TWO_ACTOR_PARAMS(-2506, -1096),      "MQ Light Trial Right Heart",                  RHT_GANONS_CASTLE_HEART,                                         RG_RECOVERY_HEART,    SpoilerCollectionCheck::RandomizerInf(RAND_INF_GANONS_CASTLE_MQ_LIGHT_TRIAL_RIGHT_HEART));
+    locationTable[RC_GANONS_CASTLE_MQ_LIGHT_TRIAL_LEFT_HEART] =               Location::Collectable(RC_GANONS_CASTLE_MQ_LIGHT_TRIAL_LEFT_HEART,               RCQUEST_MQ,      RCTYPE_FREESTANDING,                                                     ACTOR_EN_ITEM00,      SCENE_INSIDE_GANONS_CASTLE,         TWO_ACTOR_PARAMS(-2655, -549),       "MQ Light Trial Left Heart",                   RHT_GANONS_CASTLE_HEART,                                         RG_RECOVERY_HEART,    SpoilerCollectionCheck::RandomizerInf(RAND_INF_GANONS_CASTLE_MQ_LIGHT_TRIAL_LEFT_HEART));
+}
+
+static RegisterShipInitFunc initFunc(Rando::StaticData::RegisterFreestandingLocations, { "ON_BOOT" });
```

Comment:
same comment as the one on shufflefairies

```suggestion
static RegisterShipInitFunc initFunc(Rando::StaticData::RegisterFreestandingLocations);
```

Reply:
Should be fixed now